client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <Link to={`/movies/${movie.id}`}>
+            <span className="saved-movie">{movie.title}</span>
+          </Link>
         ))}
-        <div className="home-button">Home</div>
+        <Link to="/">
+          <div className="home-button">Home</div>
+        </Link>
```

Comment:
Lines 14 - 20, both of these Links wrap a single div or span element, the only purpose there is to apply a className. Take them out and put the className on Link